downgrading.md

```diff
@@ -0,0 +1,26 @@
+## Downgrading IPFS versions
+
+Downgrading your IPFS version (for instance, from 4.0 to 3.8) is not as complicated as upgrading. You can't use the same repository simultaneously for both versions. Here is how you switch
+
+First, go to your repo for IPFS, check out the right branch, and re-install.
```

Comment:
let's put this whole section under:

```
## Manual Way
```
Because https://github.com/ipfs/ipfs-update has landed and it lets you downgrade too.